prime-dotnet-webapi/Controllers/SitesController.cs

```diff
@@ -753,6 +753,32 @@ public async Task<ActionResult> SendRemoteUsersEmailUser(int siteId, IEnumerable
             return NoContent();
         }
 
+        // POST: api/Sites/5/site-reviewed-email-user
+        /// <summary>
+        /// Send site reviewed notification email to provider enrolment team
+        /// </summary>
+        /// <param name="siteId"></param>
+        /// <param name="note"></param>
+        /// <returns></returns>
+        [HttpPost("{siteId}/site-reviewed-email-user", Name = nameof(SendSiteReviewedNotificationEmail))]
```

Comment:
I assume this URL was copied from the above email endpoint? A more clear URL would just be `/site-reviewed-email`